PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -46,6 +46,7 @@ public partial class PowerShellDebuggingService : IPowerShellDebuggingService
         private Version _installedPowerShellVersion;
         private PowerShellDebuggingServiceAttachValidator _validator;
         private bool _useSSL;
+        private int _thisPid;

```

Comment:
Minor stylistic nit: avoid this as first word of variable na,e due to confusion of this.